### PR TITLE
Redirects additional details button target URL on crashes page.

### PR DESCRIPTION
**Pull request by mkarolin**
Fixes brave/brave-browser#10185

## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [x] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by rebron** [comment]
Hide for now and can do a follow-up for the github query. Thanks for the pointer @petemill

**Comment by mkarolin** [comment]
Updated PR with redirect instead of hide: @bridiver @petemill

File: browser/net/brave_common_static_redirect_network_delegate_helper.cc [hunk 1]
```diff
@@ -67,6 +97,9 @@ int OnBeforeURLRequest_CommonStaticRedirectWorkForGURL(
       URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS, kChromeCastPrefix);
   static URLPattern clients4_pattern(
       URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS, kClients4Prefix);
+  static URLPattern bugsChromium_pattern(
```
Comment (iefremov) [inline browser/net/brave_common_static_redirect_network_delegate_helper.cc:100]: bugs_chromium_pattern

**Comment by kjozwiak** [comment]
Verification **PASSED** on `macOS 10.15.5 x64` using the following build:

```
Brave | 1.12.18 Chromium: 83.0.4103.97 (Official Build) nightly (64-bit)
-- | --
Revision | 326d148b9655369b86498d9ecca39f63dd2bdd2d-refs/branch-heads/4103@{#657}
OS | macOS Version 10.15.5 (Build 19F101)
```

* ensured that both `brave://crash` and `brave://gpucrash` are appearing under `brave://crashes`
* ensured that clicking on `Provide additional details` opens GH and not 
* ensured reports are being sent when `Help improve Brave's features and performance` is enabled via `brave://settings/privacy`

Created the following crashes and ensured they appeared in `BackTrace`

* `7e590000-1c55-8704-0000-000000000000`
* `7d590000-1c55-8704-0000-000000000000`
* `80590000-1c55-8704-0000-000000000000`

As per the above, redirecting from `bugs.chromium.org` -> `GH`

chromiumLink

**Pull request merged**